src/main/tray.js

```diff
@@ -386,5 +393,14 @@ gpii.app.trayButton.windowMessage = function (that, hwnd, msg, wParam) {
         default:
             break;
         }
+    } else if (msg === that.trayButtonPositionMessage) {
+        var lParamValue = lParam.address();
+        // The bounding rectangle is 4 shorts crammed in both the message parameters.
+        that.rect = {
+            x: wParam & 0xffff,
+            y: (wParam >> 16) & 0xffff,
+            width: lParamValue & 0xffff,
+            height: (lParamValue >> 16) & 0xffff
```

Comment:
What is the idea behind this code? I see a lot of magic numbers and the shifts are a bit strange. My suggestion would be to explain this in great detail and cover fully with tests if there is no better way to write it.